spec.emu

```diff
@@ -90,29 +90,36 @@ contributors: Ujjwal Sharma, Younies Mahmoud
     </emu-table>
 
     <emu-clause id="sec-getunitoptions" aoid="GetUnitOptions">
-      <h1>GetUnitOptions ( _unit_, _options_, _baseStyle_, _prevStyle_ )</h1>
-      <p>The GetUnitOptions abstract operation is called with the arguments _unit_ (which must be a String), _options_ (which must be an Object), _baseStyle_ (which must be a String), and _prevStyle_ (which must be a String) and returns a Record containing the relevant options for that unit. The following steps are taken:</p>
+      <h1>
+        GetUnitOptions (
+          _unit_: a String,
+          _options_: an ECMAScript language value,
```

Comment:
I believe this is any ECMAScript language value, not just an Object, since it's passed straight through from a built-in.